### PR TITLE
kafka blog update

### PR DESCRIPTION
**Pull request by ashish8947**
Added kafka blog

### CODE REVIEW

File: content/blog/quick-kafka-installation/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: "Quick Kafka Installation on Windows"
```
Comment (mohammed786) [inline content/blog/quick-kafka-installation/index.md:2]: You can add title as `Setting Up and Running Apache Kafka on Windows OS`.

File: content/blog/quick-kafka-installation/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+date: "2018-08-24"
+coverImage: "messagelog.png"
+author: "Ashish Sharma"
+tags: ["Kafka"]
```
Comment (mohammed786) [inline content/blog/quick-kafka-installation/index.md:6]: you can add tags as ["Kafka", "Windows"]

File: content/blog/quick-kafka-installation/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+---
+
+
```
Comment (mohammed786) [inline content/blog/quick-kafka-installation/index.md:9]: You can also add **Prerequisite** section.

File: content/blog/quick-kafka-installation/index.md (hunk 1 above)
Comment (ashish8947) [reply content/blog/quick-kafka-installation/index.md:2]: Done

File: content/blog/quick-kafka-installation/index.md (hunk 2 above)
Comment (ashish8947) [reply content/blog/quick-kafka-installation/index.md:6]: Done

File: content/blog/quick-kafka-installation/index.md (hunk 3 above)
Comment (ashish8947) [reply content/blog/quick-kafka-installation/index.md:9]: Not Required

**Pull request merged**